canmatrix/convert.py

```diff
@@ -387,6 +388,7 @@ def main():
         # only print error messages, ignore verbosity flag
         verbosity = -1
 
+    setup_logger('root')
```

Comment:
With how `setup_logger()` is implemented now this should probably be `setup_logger(None)`.

`setup_logger()` could be adjusted to default to `None` but this can be left for another PR, or included.  Whatever.

https://docs.python.org/3/library/logging.html#logging.getLogger

So we configure the level for the root logger below maybe this becomes:

```python3
root_logger = setup_logger(None)
set_log_level(root_logger, verbosity)
```